media/css/base/mozid-modal.less

```diff
@@ -20,6 +20,7 @@ html.noscroll body {
     left: 0;
     z-index: 9999999;
     overflow: auto;
+    .animation(sand-fade-in 0.1s ease-in 0s 1 normal both);
```

Comment:
So I used an animation here, as transitions don't occur when an element changes from display:none to display:block. This method also allows older browsers to degrade gracefully.

I can adjust the animation timing :+1: 

As for the fade out, this would require a little timeout trickery before hiding the modal element again. Worth it?